app/src/main/java/com/codesoom/assignment/models/TaskManager.java

```diff
@@ -0,0 +1,107 @@
+package com.codesoom.assignment.models;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Task 의 관리를 위해 만들었습니다.
```

Comment:
```suggestion
 * 할 일을 관리합니다.
```
`Task`를 쓰게 되면 `Task` 클래스의 이름을 바꾸게 되는 일이 생길 경우 주석도 바꿔줘야 합니다. 의미를 잘 살리는 주석을 쓰되, 클래스 이름을 그대로 적는 것은 가급적 피하는 것이 좋습니다.

한편 다음과 같이 블록 코멘트 내부에 `@see` 태그를 적고 클래스 경로를 적어주면 인텔리J에서 링크가 걸려서 찾아 들어가기 쉽습니다.
```
/**
 * Task를 관리합니다.
 * 
 * @see com.codesoom.assignment.models.Task
 */
```

Reply:
JAVA doc 에 대해 좀 더 공부해보겠습니다. 감사합니다.
![image](https://user-images.githubusercontent.com/41536271/105134591-04166100-5b32-11eb-9632-77158a4c5860.png)
공식 라이브러리 들어가보니 굉장히 상세하게 적혀있군요 ㄷㄷ

Reply:
Java 기본 클래스를 틈틈이 읽어보시는 것을 권합니다. Java를 직접 만든 사람들이 쓴 코드와 주석들이기 때문에 가장 모범적인 예시입니다.